README.md

```diff
@@ -1,36 +1,56 @@
+# Form URLEncoded
 
-[![Latest release](http://img.shields.io/github/release/purescript-contrib/purescript-form-urlencoded.svg)](https://github.com/purescript-contrib/purescript-form-urlencoded/releases)
-[![Build status](https://travis-ci.org/purescript-contrib/purescript-form-urlencoded.svg?branch=master)](https://travis-ci.org/purescript-contrib/purescript-form-urlencoded)
-[![Pursuit](http://pursuit.purescript.org/packages/purescript-form-urlencoded/badge)](http://pursuit.purescript.org/packages/purescript-form-urlencoded/)
-[![Maintainer: zudov](https://img.shields.io/badge/maintainer-zudov-lightgrey.svg)](https://github.com/zudov)
-[![Maintainer: thomashoneyman](https://img.shields.io/badge/maintainer-thomashoneyman-lightgrey.svg)](https://github.com/thomashoneyman)
+[![CI](https://github.com/purescript-contrib/purescript-form-urlencoded/workflows/CI/badge.svg?branch=main)](https://github.com/purescript-contrib/purescript-form-urlencoded/actions?query=workflow%3ACI+branch%3Amain)
+[![Release](https://img.shields.io/github/release/purescript-contrib/purescript-form-urlencoded.svg)](https://github.com/purescript-contrib/purescript-form-urlencoded/releases)
+[![Pursuit](https://pursuit.purescript.org/packages/purescript-form-urlencoded/badge)](https://pursuit.purescript.org/packages/purescript-form-urlencoded)
+[![Maintainer: zudov](https://img.shields.io/badge/maintainer-zudov-teal.svg)](https://github.com/zudov)
+[![Maintainer: thomashoneyman](https://img.shields.io/badge/maintainer-thomashoneyman-teal.svg)](https://github.com/thomashoneyman)
 
-A `FormURLEncoded` datatype represents an ordered list of key-value pairs
-with possible duplicates. `encode` function allows to transform `FormURLEncoded`
-into a `Maybe String` according to `application/x-www-form-urlencoded`.
-`decode` function transforms a string into a `Maybe FormURLEncoded` structure.
+A `FormURLEncoded` datatype represents an ordered list of key-value pairs with possible duplicates. The `encode` function allows to transform `FormURLEncoded` into a `Maybe String` according to `application/x-www-form-urlencoded`. The `decode` function transforms a string into a `Maybe FormURLEncoded` structure.
 
-Documentation is available on [Pursuit][Pursuit].
+## Installation
 
-[Pursuit]: https://pursuit.purescript.org/packages/purescript-form-urlencoded
+Install `form-urlencoded` with [Spago](https://github.com/purescript/spago):
 
-Example:
+```sh
+spago install form-urlencoded
+```
+
+## Quick start
+
+Use the `encode` function to properly encode an array of key-value pairs, and the `decode` function to decode an encoded string.
 
-```haskell
+```purs
 > import Data.FormURLEncoded (fromArray, encode)
 > import Data.Maybe (Maybe(..))
 > import Data.Tuple (Tuple(..))
 
-> encode (fromArray [ Tuple "hey" Nothing
->                   , Tuple "Oh" (Just "Let's go!")
->                   ])
+> encode (fromArray [ Tuple "hey" Nothing, Tuple "Oh" (Just "Let's go!") ])
```

Comment:
Could this be changed back to the multi-line approach? I think it makes things easier to read.